File: weather-library/src/main/java/org/rcgonzalezf/weather/openweather/model/OpenWeatherCurrentData.java
package org.rcgonzalezf.weather.openweather.model;

import org.rcgonzalezf.weather.common.models.converter.Data;

public class OpenWeatherCurrentData implements Data {

  private String name;

  private String cod;

  private Wind wind;

  private java.util.List<Weather> weather;

  private Sys sys;

  private long dt;

  private Main main;

  public Wind getWind() {
    return wind;
  }

  public java.util.List<Weather> getWeather() {
    return weather;
  }

  public long getDt() {
    return dt;
  }

  public Main getMain() {
    return main;
  }

  public String getCod() {
    return cod;
  }

  public String getName() {
    return name;
  }

  public Sys getSys() {
    return sys;
  }
}

File: weather-library/src/main/java/org/rcgonzalezf/weather/openweather/model/OpenWeatherCurrentData.kt
package org.rcgonzalezf.weather.openweather.model

import org.rcgonzalezf.weather.common.models.converter.Data

class OpenWeatherCurrentData : Data {
    val name: String? = null
    val cod: String? = null
    val wind: Wind? = null
    val weather: List<Weather>? = null
    val sys: Sys? = null
    val dt: Long = 0
    val main: Main? = null
}
